packages/tradershub/src/components/Clipboard/Clipboard.tsx

```diff
@@ -50,7 +50,7 @@ const Clipboard = ({ textCopy, tooltip }: TClipboardProps) => {
             isVisible={isHovered && !isMobile}
             message={isCopied ? 'Copied!' : 'Copy'}
         >
-            <Button colorStyle='white' onClick={onClick} ref={hoverRef} size='sm'>
+            <Button colorStyle='prominent' onClick={onClick} ref={hoverRef} size='sm'>
```

Comment:
Dont think prominent exists in quill button colorStyle 
https://quill-design-6v3.pages.dev/?path=/docs/buttons-basic--docs